### ISSUE TITLE
GraphiQL / Explain Improvements

### ISSUE THREAD

**Issue by benjie** [graphile/postgraphile#1235]
- [ ] Add option to enable analyze _on queries only_
- [ ] Add option to export as JSON rather than text (for import into https://tatiyants.com/pev/#/plans/new)
- [ ] Add button (or similar) to easily copy the SQL query or explain results

**Comment by hyusetiawan** [comment]
maybe im just not used to it, but is it possible to improve the readability?
```
select to_json(dob) as "__dob"
from users as __local_0__
where (
  __local_0__.id = $1
) and (TRUE) and (TRUE)
```
is hard to parse what is going on, or perhaps there is a guide on how to understand it?

**Comment by benjie** [comment]
Related:

- https://github.com/graphile/crystal-pre-merge/issues/388
- https://github.com/graphile/crystal-pre-merge/issues/413
- https://github.com/graphile/crystal/pull/1618

**Comment by benjie** [comment]
@hyusetiawan FYI the SQL in V5 is significantly easier to understand:

# V4

![image](https://github.com/graphile/crystal/assets/129910/6f893e16-f11a-4c10-a5cc-159ff2f5040a)

# V5

![image](https://github.com/graphile/crystal/assets/129910/39cfc3cd-08de-4c66-b0ed-e63882ade352)
